src/test/scala-2.11/org/apache/james/gatling/jmap/scenari/JmapAllScenario.scala

```diff
@@ -0,0 +1,34 @@
+package org.apache.james.gatling.jmap.scenari
+
+import io.gatling.core.Predef._
+import org.apache.james.gatling.control.UserCreator
+import org.apache.james.gatling.jmap.{JmapMessages}
+import org.apache.james.gatling.jmap.scenari.common.Configuration._
+import org.apache.james.gatling.jmap.scenari.common.{CommonSteps, HttpSettings}
+
+import scala.concurrent.duration._
+
+class JmapAllScenario extends Simulation {
+  private val loopVariableName = "any"
+
+  val users = new UserCreator(BaseJamesWebAdministrationUrl).createUsersWithInboxAndOutbox(UserCount)
+
+  val scn = scenario("JmapAllScenarios")
+    .exec(CommonSteps.provisionSystemMailboxes(users))
+    .during(ScenarioDuration) {
+      exec(JmapMessages.sendMessagesRandomlyWithRetryAuthentication(users))
+      .pause(1 second, 5 seconds)
+      .exec(JmapMessages.listMessagesWithRetryAuthentication())
+      .pause(1 second, 5 seconds)
+      .exec(JmapMessages.getMessagesWithRetryAuthentication())
+      .pause(1 second, 5 seconds)
```

Comment:
I guess that's not the pausing that is issuing Inbox: if you look at your browser network debug tag, you'll see that they often send requests 3 at a time (getMailboxes + getMessageList + getMessages). Idem when sending a message where getMailboxes is called just after setMessages.